tests/test_executions.py

```diff
@@ -9,13 +11,15 @@
 
 
 class TestRegistry(RQTestCase):
+    """Test the execution registry."""
+
     def setUp(self):
         super().setUp()
+        self.q = Queue(connection=self.connection)
```

Comment:
Nitpick: mind changing `self.q` to `self.queue`? I want to keep things consistent whenever possible.